packages/playground/cases/context/delete-file/src/index.js

```diff
@@ -5,6 +5,7 @@ console.log(context);
 document.getElementById("root").textContent = "__PAGE_RENDER__";
 
 if (module.hot) {
+	module.hot.accept();
```

Comment:
Why changed this?

Reply:
This test case was added by myself in pull request #3749 , I forgot to add module.hot.accept. The context module did not change if the code generation key hashed by module identifier at that time. 

But now it is hashed by module graph and exports info, so the code generation result will change because of the file deleting. So if it is not self accepted, the page will be refresh and test case will be suspended until the timeout.